aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -1866,6 +1867,38 @@ impl VMValidator for AptosVM {
             }
         }
 
+        if !self
+            .vm_impl
+            .get_features()
+            .is_enabled(FeatureFlag::WEBAUTHN_SIGNATURE)
+        {
+            let is_webauthn_signature = |signature: &AnySignature| -> bool {
+                if let AnySignature::WebAuthn { .. } = signature {
+                    return true;
+                }
+                false
+            };
+
+            let sender = transaction.authenticator_ref().sender();
```

Comment:
I don't think this will be enough because there could be WebAuth authenticators on `secondary_signers()` and on `fee_payer()` too. cc @lightmark

Reply:
I think `sender()` here returns an `AccountAuthenticator` 

code reference: https://github.com/aptos-labs/aptos-core/blob/6a567790fe1ecce16ee3e7fa0f16953afa7edcde/types/src/transaction/authenticator.rs#L214

Reply:
...yep, `secondary_signers()` and `fee_payer()` also return `AccountAuthenticator`'s

Reply:
Fixed